blockchain/state/sync_test.go

```diff
@@ -220,17 +220,18 @@ func TestIterativeStateSyncBatched(t *testing.T)    { testIterativeStateSync(t,
 
 func testIterativeStateSync(t *testing.T, count int) {
 	// Create a random state to copy
-	srcDb, srcRoot, srcAccounts := makeTestState(t)
+	srcState, srcRoot, srcAccounts := makeTestState(t)
 
 	// Create a destination state and sync with the scheduler
-	dstDb := database.NewMemoryDBManager()
-	sched := NewStateSync(srcRoot, dstDb, statedb.NewSyncBloom(1, dstDb.GetMemDB()), nil)
+	dstDiskDb := database.NewMemoryDBManager()
+	dstState := NewDatabase(dstDiskDb)
+	sched := NewStateSync(srcRoot, dstDiskDb, statedb.NewSyncBloom(1, dstDiskDb.GetMemDB()), nil)
```

Comment:
Just a minor question, however, what `sched` stands for?

Reply:
It is stateSync object, `sched` word is used also in `fasysync`.